blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc

```diff
@@ -0,0 +1,41 @@
+:title: Meet Keycloak at KubeCon Salt Lake City, Utah in Nov 2024
+:date: 2024-04-10
+:publish: true
+:author: Kamesh Akella
+
+We are thrilled to announce that Keycloak will be at KubeCon Salt Lake City, Utah in Nov 2024. There are several Keycloak specific sessions lined up during this Conference, and we will be hosting a Kiosk at the Project Pavilion at KubeCon NA 2024.
```

Comment:
I hear at least two people from Hitachi plan to join us at the Kiosk. So I'd like to keep it a bit more open. Maybe we can reach out to them to have them listed here as well. 

```suggestion
We are thrilled to announce that Keycloak will be at KubeCon Salt Lake City, Utah in Nov 2024. There are several Keycloak specific sessions lined up during this conference, and we will be hosting a Kiosk at the Project Pavilion at KubeCon.
```

Reply:
Good Idea @ahus1 I will reach out to them and find the names if possible.